spec/index.bs

```diff
@@ -1842,6 +1842,21 @@ interface GPUQueue {
     GPUFence createFence(optional GPUFenceDescriptor descriptor = {});
     void signal(GPUFence fence, unsigned long long signalValue);
 
+    void setBufferSubData(
+        GPUBuffer destination,
+        GPUBufferSize destinationOffset,
+        GPUBufferSize size,
+        ArrayBuffer sourceData,
+        GPUBufferSize sourceOffset);
+
+    void setTextureSubData(
```

Comment:
bikeshed:writeTexture? (this was suggested some time ago offline by @beaufortfrancois )